src/goto-instrument/contracts/instrument_spec_assigns.h

```diff
@@ -203,10 +198,12 @@ class instrument_spec_assignst
   instrument_spec_assignst(
```

Comment:
Please also add a `\param` comment above for `_cfg_info`.

Reply:
thanks